group_project_v2/group_project.py

```diff
@@ -531,20 +541,22 @@ def submissions_view(self, context):
 
         return fragment
 
-    def _render_dashboard_view(self, context, view):
-        fragment = Fragment()
-
-        children_context = context.copy()
-        self._append_context_parameters_if_not_present(children_context)
-
-        stage_fragments = self._render_children(view, children_context, self.stages)
-        stage_contents = [frag.content for frag in stage_fragments]
-        fragment.add_frags_resources(stage_fragments)
-
-        render_context = {'activity': self, 'stage_contents': stage_contents}
-        fragment.add_content(self.render_template(view, render_context))
-
-        return fragment
+    # TODO: This seems to be a dead code, so I'll remove it
```

Comment:
@jbzdak please don't comment dead code - remove it; we have a source control to keep history and restore it if need emerges.